tests/scripts/minikube.sh

```diff
@@ -89,27 +89,29 @@ function check_context() {
   return 1
 }
 
-# configure dind-cluster
-KUBE_VERSION=${KUBE_VERSION:-"v1.8.0"}
+# configure minikube
+KUBE_VERSION=${KUBE_VERSION:-"v1.9.1"}
+MEMORY=${MEMORY:-"3000"}
 
 case "${1:-}" in
   up)
-    minikube start --memory=3000 --kubernetes-version ${KUBE_VERSION} --extra-config=apiserver.Authorization.Mode=RBAC
-    wait_for_ssh
-
+    # Use kubeadm bootstrapper for 1.9+ since localkube was deprecated in 1.8
+    if [[ $KUBE_VERSION == v1.5* ]] || [[ $KUBE_VERSION == v1.6* ]] || [[ $KUBE_VERSION == v1.7* ]] ; then
+      echo "starting minikube with localkube bootstrapper"
+      minikube start --memory=${MEMORY} --kubernetes-version ${KUBE_VERSION} --extra-config=apiserver.Authorization.Mode=RBAC
+      wait_for_ssh
+      enable_roles_for_RBAC
+      echo "initializing flexvolume for rook"
+      init_flexvolume
+    else
+      echo "starting minikube with kubeadm bootstrapper"
+      minikube start --memory=${MEMORY} -b kubeadm --kubernetes-version ${KUBE_VERSION} --extra-config=kubelet.node-ip=192.168.99.100
```

Comment:
@jbw976 The extra kubelet config is so that we can run the smoke test and get it to work. Using kubeadm bootstrapper, the iso setups two interfaces. One is used as internal and the other one as external. Without this, the test will fail trying to reach the Rook-api because the pod gets assigned the internal IP.

I am not too fold of this and it looks hackish. We could remove it, and when we someone runs the smoke tests, they can pass that variable. Ill let you guys decide.